open_ce/images/builder-cuda-ppc64le/Dockerfile.cuda-10.2

```diff
@@ -48,4 +48,5 @@ RUN export PATH="${PATH}" && \
     sudo rm -f /run/nologin && \
     yes | ssh-keygen -f $HOME/.ssh/id_rsa -q -t rsa -N '' && cp $HOME/.ssh/id_rsa.pub $HOME/.ssh/authorized_keys && \
     echo ". $CONDA_HOME/etc/profile.d/conda.sh" >> ${HOME}/.bashrc && \
+    echo "export PYTHONPATH=${PYTHONPATH}:$HOME/open_ce" >> ${HOME}/.bashrc && \
```

Comment:
Would it be better to pip install the package, instead of setting the python path in the bashrc file?

That way it's being used more as intended?

Reply:
We discussed this and decided to keep it as it is as of now.